worker/process-image.js

```diff
@@ -96,6 +105,31 @@ function uploadToS3 (s3, path, key, metadata, thumbPath, callback) {
   });
 }
 
+function translateJPEG2000 (ext, path, key, callback) {
+  if (ext === '.jp2') {
+    log(['debug'], 'Converting JPEG2000 to TIFF.');
+    var parsedPath = pathTools.parse(path);
+    var outPath = pathTools.join(parsedPath.dir, parsedPath.name) + '.tif';
+
+    var cmd = `${translateExe} -of GTiff ${path} ${outPath} ` +
+              '-co TILED=yes ' +
+              '-co COMPRESS=DEFLATE ' +
+              '-co PREDICTOR=2 ' +
+              '-co SPARSE_OK=yes ' +
+              '-co BLOCKXSIZE=256 ' +
+              '-co BLOCKYSIZE=256 ' +
+              '-co INTERLEAVE=band ' +
+              '-co NUM_THREADS=ALL_CPUS';
+    cp.execSync(cmd);
```

Comment:
We could add a test to see whether the file exists after the command completes. I did try to set this up with cp.exec, the asynchronous version of this command which follows the node convention
```js
exec('cmd', (error, stdout, stderr) => {
  if (error) ...
}
```
but since gdal_translate doesn't seem to return stdout I wasn't sure how to wait for the command to complete before continuing. I'm pretty inexperienced with async node though; do you know how we could handle this?